cli_tools/import_precheck/check_disks_linux.go

```diff
@@ -60,47 +87,114 @@ func (c *disksCheck) getName() string {
 	return "Disks Check"
 }
 
-func (c *disksCheck) run() (*report, error) {
-	r := &report{name: c.getName()}
-	var out string
-	var err error
-	if c.lsblkOverride != nil {
-		out, err = c.lsblkOverride()
-	} else {
-		out, err = c.lsblk()
+// mountPoints supports listing the mounts on a system, and determining which block device(s)
+// the mount is physically contained. This is helpful since technologies such as LVM allow
+// logical mounts to span physical devices.
+type mountPoints struct {
+	mounts []struct {
+		dir       string
+		hierarchy []string
 	}
-	if err != nil {
-		return nil, err
+}
+
+// listPhysicalDevicesForMount returns the the block device(s) that contain the mount.
+// An empty return value means the mount was not found.
+func (m *mountPoints) listPhysicalDevicesForMount(mount string) (devices []string) {
+	set := map[string]struct{}{}
+	for _, mp := range m.mounts {
+		if mp.dir == mount {
+			set[mp.hierarchy[0]] = struct{}{}
```

Comment:
Why only [0]?

Reply:
Good question! The first entry is the physical device; added a function and docs to help explain.